peps/pep-0757.rst

```diff
@@ -183,9 +183,9 @@ create a Python :class:`int` object from a digits array.
    greater than or equal to 0.
 
    The caller must initialize the array of digits *digits* and then call
-   :c:func:`PyLongWriter_Finish` to get a Python :class:`int`. Digits must be
-   in the range [``0``; ``PyLong_BASE - 1``]. Unused digits must be set to
-   ``0``.
+   :c:func:`PyLongWriter_Finish` to get a Python :class:`int`.  Digits must be
+   in the range [``0``; ``(1 << sys.bits_per_digit) - 1``].  Unused digits must
```

Comment:
`(1 << sys.bits_per_digit)` is not easily accessible in C. What's wrong with PyLong_BASE? It's a public constant.

Reply:
`sys.bits_per_digit` doesn't  exist, I suppose that you mean `sys.int_info.bits_per_digit`.